docs/platform/howto/manage-org-vpc-peering-aws.md

```diff
@@ -58,24 +58,22 @@ create an organization VPC peering connection using a tool of your choice:
    1. Click **Create**.
 
 </TabItem>
-<!--
 <TabItem value="cli" label="Aiven CLI">
 
 Run the `avn organization vpc peering-connection create` command:
 
 ```bash
-avn organization vpc peering-connection create \
-  --organization-id AIVEN_ORGANIZATION_ID      \
-  --project-vpc-id AIVEN_ORGANIZATION_VPC_ID   \
-  --peer-cloud-account AWS_ACCOUNT_ID          \
+avn organization vpc peering-connection create    \
+  --organization-id AIVEN_ORGANIZATION_ID         \
+  --organization-vpc-id AIVEN_ORGANIZATION_VPC_ID \
```

Comment:
I can't help noticing that in some examples the placeholders for values are like `$vpc_id` and in others like this here. Consistency would make them look a bit better I think.